AUTHORS.rst

```diff
@@ -64,3 +64,14 @@ The following people have been core contributors to scikit-learn's development a
   * Nelle Varoquaux
   * `Gael Varoquaux <http://gael-varoquaux.info/blog/>`_
   * Ron Weiss
+
+Please do not email the authors directly with bug reports or feature requests.
+To address those concerns, please make use of the
```

Comment:
To avoid duplicated information, I would just add a link to the "What's the best way to ask questions about scikit-learn" section in the FAQ.